api/index.js

```diff
@@ -1,49 +1,34 @@
-require('dotenv').config()
 const { SESSION_NAME, SESSION_MAX_AGE, SECRET, SITE_URL } = process.env
 
-const { send } = require('micro')
-const morgan = require('micro-morgan')
-const redirect = require('micro-redirect')
 const session = require('micro-cookie-session')({
```

Comment:
Was going to replace this library as we'll use the native zeit now request handling, but left it in there as I couldn't find a suitable replacement. Haven't been able to verify that it works as intended though